Automattic/wp-calypso
# Framework: Remove legacy notices

#### Changes proposed in this Pull Request

This PR removes the legacy notices `EventEmitter` store. An end of an era. See #48408.

#### Testing instructions

* Go through Calypso as a logged in, and logged out user and do some smoke testing.
* Verify global notices still work well.
* Go to `/devdocs/design/global-notices` and verify the example still works well.

Fixes #48408.
